zad1/main.rb

```diff
@@ -1,13 +1,13 @@
 def sort_letters(string)
-
+  string.reverse
 end
 
 def count_vowels(string)
-
+  string.count 'a e i o u y'
```

Comment:
spacje są tu zbędne, wręcz wywołuje błąd w Twoim rozwiązaniu, bo poza samogłoskami liczy też spacje. Ogólnie nie to było esencją zadania żeby patrzeć na spację w stringu w argumencie, no ale jednak tej metodzie nie podaje się spacji, chyba ze naprawdę chcemy